package/MDAnalysis/analysis/hbonds/hbond_analysis.py

```diff
@@ -281,14 +290,16 @@ class HydrogenBondAnalysis_OtherFF(HydrogenBondAnalysis):
           0. "time"
           1. "donor_idx"
           2. "acceptor_idx"
-          3. "donor_resnm"
-          4. "donor_resid"
-          5. "donor_atom"
-          6. "acceptor_resnm"
-          7. "acceptor_resid"
-          8. "acceptor_atom"
-          9. "distance"
-          10. "angle"
+          3. "donor_idx_zero"
```

Comment:
I don't like the "zero" in something that will eventually become the primary name. Can we go with `donor_index` and `acceptor_index`? I know that this is more confusing if you just look at the names but it will be better once we remove `donor_idx` and `acceptor_idx`.